### DISCUSSION TITLE
Discussion on Repo 🥳🥳

### DISCUSSION THREAD

**Discussion by Ishan-co** [ishan-open/inext-fronts#1] [Announcements]
صرفا نیاز بود اعلام کنیم که روی ریپو دیسکاشن رو فعال کردم که بتونیم ایده ها و مطالب رو توی ایشو نذاریم!!

**Comment by Ishan-co** [comment]
@amirHossein-Ebrahimi 
@alimohammad0816

**Reply by realamirhe** [reply]
🎉🎉

**Reply by alimohammad0816** [reply]
🎉🎉🎉
@amirHossein-Ebrahimi
